syntax = "proto2";

package ssd.protos;

import "ssd/protos/anchor_list.proto";
import "ssd/protos/train.proto";
import "ssd/protos/eval.proto";

message Model {
    required string name = 1;
    required string type = 2 [default="ALEX"];
    required int32 num_classes = 3;
    optional string npy_path = 4 [default=""];

    required AnchorList anchor_list = 5;
    required Train train = 6;
    required Eval eval = 7;
}